simulation/utils/generate_map.py

```diff
@@ -59,42 +48,41 @@ def get_seed(self):
 
     def get_X_list(self):
         """Get list of x coordinates"""
-        if not len(self.map_points):
+        if self.map_points.size == 0:
             xs = [point.x for point in self.points_before_interpolation]
         else:
-            xs = [point[0] for point in self.map_points]
-
+            xs = self.map_points[:, 0]
         return xs
 
-    
     def get_Y_list(self):
         """Get list of y coordinates"""
-        if not len(self.map_points):
+        if self.map_points.size == 0:
             ys = [point.y for point in self.points_before_interpolation]
         else:
-            ys = [point[1] for point in self.map_points]
-
+            ys = self.map_points[:, 1]
         return ys
 
-    
     def __len__(self):
         """ Length of map (points, not segments) """
-        if len(self.map_points):
+        if self.map_points.size == 0:
             return len(self.points_before_interpolation)
         else:
             return len(self.map_points)
-        
+
     def __delitem__(self, index):
-        if not len(self.map_points):
+        if self.map_points.size == 0:
             del self.points_before_interpolation[index]
         else:
             del self.map_points[index]
 
     def are_points_in_clockwise_order(self, A, B, C):
         return (C.y - A.y) * (B.x - A.x) < (B.y - A.y) * (C.x - A.x)
 
-    def are_lines_intersecting(self, i, j):
-        """Function that checks if two lines ((X[i],Y[i]), (X[i+1], Y[i+1])), ((X[j],Y[j]), (X[j+1],Y[j+1])) intersect"""
+    def are_lines_intersecting(self, index_of_first_line_starting_point, index_of_second_line_starting_point):
+        """Function that checks if two lines intersect"""
+        i = index_of_first_line_starting_point
+        j = index_of_second_line_starting_point
```

Comment:
Nie do końca o to mi chodziło, ale tak też jest git XD

Reply:
mi zależało, żeby te kolejne linijki nie były za długie, więc poszłam na kompromis xd